book/website/ethical-research/activism/activism-env-impact.md

```diff
@@ -0,0 +1,166 @@
+
+The rise in this global average temperature is strikingly illustrated by the ‘Warming Stripes’ (‘#ShowYourStripes’) project, a minimalist-style data visualisation created by Ed Hawkins. Temperatures are indicated as stripes in blue hues, representing cooler temperatures, or red hues, representing warmer ones. The general trend shows a progression from there being more blue on the left (towards the past) to more red on the right (towards the present). In short, the planet is, overall, warming!
+
+```{figure} ../../figures/global_warming.*
+---
+height: 500px
+name: Global warming stripes, by Ed Hawkins.
+alt: Global warming stripes, by Ed Hawkins.
+---
+Global warming stripes, by Ed Hawkins.
+```
+
+
+(er-activism-env-impact-problems-from-rising-temp)=
+### Problems from such rising temperatures
+
+Why should we be concerned about this rise in temperature?
+
+... TBC!
+
+(er-activism-env-impact-awareness-raising)=
+## Awareness raising
+
+Below we share some thought-provoking questions about the Environmental Impact of Digital Research.
+
+- Where in the world did you last computer end? An estimated 50 million tons of electronic waste are discarded each year. This is equivalent to throwing out 1000 laptops every single second. Less than 20% of e-waste is formally recycled, with 80% either ending up in landfill on developing countries, exposing vulnerable populations and their environment to health and pollution impacts (ref: https://www3.weforum.org/docs/WEF_A_New_Circular_Vision_for_Electronics.pdf).
+- How many unused or unaccessible data do you have on the cloud? In 2021, on average, 35% of enterprise data is “dark,” meaning it has an unknown value, while 50% is redundant, obsolete or trivial. Only about 16% is business-critical, the study concluded. (ref: https://www.veritas.com/content/dam/Veritas/docs/reports/GA_ENT_AR_Veritas-Vulnerability-Gap-Report-Global_V1414.pdf)
+- How much of the world's total electricity the technological sector will use by 2025? It is predicted that by 2025, the technology sector will consume 20% of the world’s total electricity (compared to 7% in 2022). Ref: https://www.tier1.com/the-environmental-impact-of-our-data-storage/)
+
+(er-activism-env-impact-reduction-strategies)=
+## Strategies for reduction
+
+There are several things that can be done to reduce the environmental impact of digital research. In this section we describe some of the possible actions grouped in two major groups, computing and data, and links to external resources that can implement them.
+
+These actions are not listed in any order of importance, and implementing any or all of them is better than not doing anything at all.
+
+You can also see Ten simple rules to make your computing more environmentally sustainable {cite:ps}`lannelongue2021ten` and the [Digital Humanities Climate Coalition Toolkit](https://sas-dhrh.github.io/dhcc-toolkit/).
+
+(er-activism-env-impact-computing)=
+### Computing 

```

Comment:
I'd suggest to add a short intro of computing-related impacts.

Reply:
Good idea -- if you could suggest some text that would be great (otherwise I will try to have a go next week)

Reply:
What about something like:

As we engage in research activities, our utilization of computers and code execution significantly affects the environment. In this section, we explore approaches that can be employed to mitigate the environmental impact arising from the computational aspects of our research endeavors.